### PR TITLE
Disposition page site tour POC

### PR DESCRIPTION
**Pull request by kewitham**
[Spike: find site tour package](https://github.com/usdoj-crt/crt-portal-management/issues/1657)
[Disposition Page - In-app onboarding flow#1667](https://github.com/usdoj-crt/crt-portal-management/issues/1667)

## What does this change?

This PR adds a site tour package called [shepherd js](https://shepherdjs.dev/) to guide users through the new disposition page.

## Screenshots (for front-end PR):
Image

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for, document, and establish a testing plan for any behavior that may vary across environments or is otherwise difficult to test.
+ [ ] Check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.).
+ [ ] Check for any behavior that may vary across environments or is difficult to test, and ensure that it is well-understood, documented, and that there is a testing plan in place.
+ [ ] Re-check for [accessibility](https://github.com/usdoj-crt/crt-portal/blob/develop/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/static/js/tour.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        text: 'Next',
+        action: tour.next
+      }
+    ]
+  });
+
+  tour.addStep({
+    id: 'fourth-step',
+    text:
+      'The <b>other scheduled reports</b> tab shows reports that have expiration dates more than 30 days in the future.',
```
Comment (okkays) [inline crt_portal/static/js/tour.js:67]: (prefer `strong` to `b` for [accessibility reasons](https://stackoverflow.com/a/271776/13231999))

File: crt_portal/static/js/tour.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    classes: 'sixth-step',
+    buttons: [
+      {
+        text: 'Done',
+        action: tour.next
+      }
+    ]
+  });
+
+  if (!localStorage.getItem('shepherd-tour')) {
```
Comment (okkays) [inline crt_portal/static/js/tour.js:115]: nit: setting shepherd-tour to `'no'` will still suppress the tour (which is probably fine)

File: crt_portal/static/vendor/shepherd.js
Comment (okkays) [inline crt_portal/static/vendor/shepherd.js:1]: sidenote, possible for a future tech task: is it necessary to include this in `vendor` even if we have it in package.json? `gulp` ought to be able to copy the vendor javascript to the correct directory following `npm install`

**Review by okkays — CHANGES_REQUESTED** [review]
shepherd-tour

Because of the redirect when I go to the disposition page, the site tour appears momentarily and then is skipped on refresh.

---

Also, some weird stuff is happening when I scroll down during the tour - it looks like maybe it's trying to highlight the tabs?

image

File: crt_portal/static/vendor/shepherd.js
Comment (kewitham) [reply crt_portal/static/vendor/shepherd.js:1]: created an [issue](https://github.com/usdoj-crt/crt-portal-management/issues/1679) in the backlog for this

**Review by okkays — APPROVED** [review]
LGTM - ready to demo!

**Pull request merged**